pkg/catalog/catalog.go

```diff
@@ -425,9 +425,9 @@ func BuildQueryResultMetaPath(accountName, statementId string) string {
 	return fmt.Sprintf(QueryResultMetaPath, accountName, statementId)
 }
 
-func BuildQueryResultMetaName(accountName, statementId string) string {
-	return fmt.Sprintf(QueryResultMetaName, accountName, statementId)
-}
+//func BuildQueryResultMetaName(accountName, statementId string) string {
```

Comment:
If you do not need these code, why not just remove them? The same as following.

Reply:
remove done